src/client/Microsoft.Identity.Client/TenantProfile.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Security.Claims;
+using Microsoft.Identity.Client.Utils;
+using Microsoft.Identity.Json;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Contains information of a single account. A user can be present in multiple directories and thus have multiple accounts.
+    /// This information is used for token cache lookup and enforcing the user session on the STS authorize endpoint.
+    /// </summary>
+    internal sealed class TenantProfile : ITenantProfile
+    {
+        private readonly string _idToken;
+
+        public TenantProfile(string oid, string tenantId, string idToken, bool isHomeTenant)
+        {
+            Oid = oid;
+            TenantId = tenantId;
+            _idToken = idToken;
+            IsHomeTenant = isHomeTenant;
+            Claims = GetClaimsFromIdToken();
+        }
+
+        public string Oid { get; }
+
+        public string TenantId { get; }
+
+        public ClaimsPrincipal Claims { get; }
+
+        public bool IsHomeTenant { get; }
+
+        public override string ToString()
+        {
+            return string.Format(
+            CultureInfo.CurrentCulture,
+            "Tenant Id {1}", TenantId);
```

Comment:
>enant Id {1} [](http://example.com/codeflow?start=14&length=12)

should be `{0}` and is it used anywhere? if for logging, tenant Id is PII.

Reply:
It is not used anywhere. I think I can remove this. Thanks